tests/framework/util.go

```diff
@@ -134,3 +137,20 @@ func RemoveDataVolume(dvName string, namespace string) {
 	err = virtCli.CdiClient().CdiV1alpha1().DataVolumes(namespace).Delete(dvName, nil)
 	Expect(err).ToNot(HaveOccurred())
 }
+
+func GetLatestGitHubReleaseURL(user_name string, repo_name string) string {
```

Comment:
I could see a version in pure go, but I do not require it if this is faster.

However, if this is used for downloading the latest templates then I think we need to change that decision. Are the kubevirt-ansible tests executed on top of CNV as installed using kubevirt-ansible? Because the templates are installed by kubevirt-ssp role from kubevirt-ansible repo: https://github.com/kubevirt/kubevirt-ansible/blob/master/roles/kubevirt-ssp/files/common-templates-v0.4.1.yaml

So the function to download latest github stuff might be ok functionality wise, but I am not sure kubevirt-ansible should be downloading anything. The kubevirt-ansible repo already contains all the files that should be tested.